Sources/Client/Client/ClientLogger.swift

```diff
@@ -125,18 +129,27 @@ public final class ClientLogger {
         }
     }
     
+    /// Controls whether to display icons in logs
+    /// Only valid when default `ClientLogger.logger` block is used, if you've overridden that, this is not valid.
+    public static var iconEnabled = true
```

Comment:
I'd suggest setting this value to `false` by default since the task was about removing the emojis from the logs.

Reply:
But...but....I love them!!!
Should we disable them?

Reply:
I believe the default should be `false`. They look a little bit childish, sorry man.

Reply:
No, please. We have it by default. I add 👍 to keep it enabled by default.

Reply:
They are for readability, visually grouping by icons.

Reply:
and a 👎 for no emojis ;)

Reply:
It's started... democracy or something? 😄